docs/content/functioneel/informatieobjecten_api.md

```diff
@@ -0,0 +1,78 @@
+| *Wijzigen*     | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.         |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.         |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Geen voorwaarden.  |
+| - gevolg       | Een overzicht van in de Documentregistratiecomponent opgeslagen informatieobjecten wordt geretourneerd.|
+| Specificaties  | https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/enkelvoudiginformatieobject_list   | 
+| *Raadplegen details*   |      |
+| - voorwaarde   | Bij het opvragen van een specifiek enkelvoudg informatieobject moet de identificatie van het informatieobject door de consumer aangeleverd worden.     |
+| - gevolg       | Een enkelvoudg informatieobject geïdentificeerd door de aangeleverde identificatie wordt geretourneerd. |
+| Gegevens       | Objecttype ENKELVOUDIGINFORMATIEOBJECT inclusief relatie naar INFORMATIEOBJECTTYPE.<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/enkelvoudiginformatieobject_read   | 
+| Samenhang      | - |
+| Bijzonderheden | Enkelvoudig Informatieobject is een subtype van Informatieobject. Beide objecttypen zijn "platgeslagen" zodat met het opvragen van een enkelvoudiginformatieobject ook de eigenschappen van het type informatieobject geretourneerd worden.
+|
+
+
+
+### Resource: Objectinformatieobjecten
+
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil objectinformatieobjecten opvragen uit of een objectinformatieobject toevoegen aan de documentregistratiecomponent DRC.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | Zowel de verwijzing (URI) waarmee de zaak als de verwijzing (URI) waarmee het informatieobject geïdentificeerd kunnen worden moeten aangeleverd worden door de consumer.         |
+| - gevolg       | Na het toevoegen van een objectinformatieobject is het informatieobject gekoppeld aan een zaak.|
+| Specificaties  | https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/objectinformatieobject_create   |
+| *Wijzigen*     | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.         |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.         |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
```

Comment:
Ik mis hier dat je kan filteren op:

* `object`: opvragen alle informatieobjecten die gerelateerd zijn het object (de zaak of het besluit)
* `informatieobject`: opvragen alle objecten die gerelateerd zijn aan het informatieobject

Reply:
FYI - https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/objectinformatieobject_list -> "Query parameters"

Reply:
Ik heb het aangepast/uitgebreid.